kong/conf_loader/init.lua

```diff
@@ -961,6 +961,12 @@ local function load(path, custom_conf, opts)
     end
   end
 
+  -- TODO: remove this when cluster_rpc is ready for GA
+  if conf.cluster_rpc then
```

Comment:
Why do we need to turn it off in the code instead of relying on the cluster_rpc configuration?

Reply:
It is a warning message, not turn off.